tests/conftest.py

```diff
@@ -61,6 +64,11 @@ def post_request_valid_signature():
     return request_data
 
 
+@pytest.fixture()
+def mocked_valid_signature(mocker):
+    return mocker.patch("lambdas.webhook.valid_signature", return_value=True)
```

Comment:
In light of our conversation yesterday, this continues to make sense to me.

If we know the webhook handler function `lambdas.webhook.valid_signature()` is hashing the request body + `ALMA_CHALLENGE_SECRET`, and there _are_ specific tests exercising that `lambdas.webhook.valid_signature()`, it seems safe to mock that response when testing other parts of the webhook handing.